### PR TITLE
Feat/pnl highchart

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Changed Crab V2 Pnl graph library 
- Crab/Eth series 
- Crab/Usd
- Eth/Usd

Fixes ENG-558

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 1]
```diff
@@ -1,146 +1,129 @@
...
+      yAxis: 1,
+      data: pnlSeries,
+      tooltip: {
+        valueDecimals: 2,
+        valueSuffix: '%',
+      },
+    }
+    ,{
+      yAxis: 1,
+      name: 'Crab/Eth',
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:72]: CRAB/ETH Return

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 2]
```diff
@@ -1,146 +1,129 @@
...
+      data: crabEthSeries,
+      tooltip: {
+        valueDecimals: 6,
+        valueSuffix: ' ETH'
+      },
+      color: "#F5B7B1"
+    },
+    {
+      yAxis: 0,
+      name: 'Crab/Usd',
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:82]: CRAB/USD % Return

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 3]
```diff
@@ -1,146 +1,129 @@
...
+      data: crabUsdSeries,
+      tooltip: {
+        valueDecimals: 2,
+        valuePrefix: '$'
+      },
+      color: "#21618C"
+    }
+    ,{
+      yAxis: 0,
+      name: 'Eth/Usd',
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:92]: ETH/USD % return

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 4]
```diff
@@ -1,146 +1,129 @@
...
+    },
+    {
+      yAxis: 0,
+      name: 'Crab/Usd',
+      data: crabUsdSeries,
+      tooltip: {
+        valueDecimals: 2,
+        valuePrefix: '$'
+      },
+      color: "#21618C"
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:88]: Think we should make this a much bolder color. Crab aims to make USD returns, so imo the USD return line needs to stand out the most. 

bolder color would be good 

also the Crab/USD line being on top of all the other lines

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 5]
```diff
@@ -1,146 +1,129 @@
...
+    }
+    ,{
+      yAxis: 1,
+      name: 'Crab/Eth',
+      data: crabEthSeries,
+      tooltip: {
+        valueDecimals: 6,
+        valueSuffix: ' ETH'
+      },
+      color: "#F5B7B1"
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:78]: Would make this color less bold, Crab / ETH returns there for comparison / as a benchmark but less something to draw attention to

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx [hunk 6]
```diff
@@ -1,146 +1,129 @@
...
+    useEffect(() => {
+      Highcharts.setOptions({
+       lang: {
+        thousandsSep: ','
+       }
+      });
+     }, []);
+
+    const series = [{
+      name: 'PnL',
```
Comment (alexisgauba) [inline packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx:62]: we can remove this once we make the below 3 % returns, so the graph should only have 3 lines

**Review by alexisgauba — CHANGES_REQUESTED** [review]
Think we should show CRAB/ETH CRAB/USD and ETH/USD as % returns just like Ammy's graphs which people tend to grok the most https://twitter.com/opyn_/status/1575856155305222144/photo/1

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx (hunk 1 above)
Comment (DemolaJames) [reply packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx]: Updated

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx (hunk 3 above)
Comment (DemolaJames) [reply packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx]: Updated

File: packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx (hunk 4 above)
Comment (DemolaJames) [reply packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx]: Done

**Review by alexisgauba — APPROVED** [review]
lgtm 🦀

**Review by KMKoushik — APPROVED** [review]

**Pull request merged**